bin/lib/gameLonger.js

```diff
@@ -83,10 +85,7 @@ class Game{
 		const missing_fields = [];
 
     // handle when we are re-building a Game instance from a simple obj (e.g. from the DB)
-		if( config === undefined ) {
-			GAMES_STATS.counts.created += 1;
-		} else {
-			GAMES_STATS.counts.cloned += 1;
+		if( config !== undefined ) {
 			if (userUUID !== config['uuid']) {
 				throw `Game.constructor: config defined, but mistmatched uuids: userUUID=${userUUID}, config.uuid=${config.uuid}, config=${JSON.stringify(config)}`;
```

Comment:
Is this ever caught anywhere? From what I can see, this isn't in a promise chain - throwing the error could bring down the server. Is that intentional?